### PR TITLE
[mod_articles_category]Code style

### PR DESCRIPTION
**Pull request by wojsmol**
### Summary of Changes

Code style changes for mod_articles_category
### Testing Instructions

code review

cc @JoomliC @andrepereiradasilva

### CODE REVIEW

File: modules/mod_articles_category/mod_articles_category.xml [hunk 1]
```diff
@@ -28,7 +28,7 @@
 					default="normal"
 					label="MOD_ARTICLES_CATEGORY_FIELD_MODE_LABEL"
```
Comment (andrepereiradasilva) [inline modules/mod_articles_category/mod_articles_category.xml]: move default down?

File: modules/mod_articles_category/mod_articles_category.xml [hunk 2]
```diff
@@ -77,135 +80,145 @@
...
 					<option value="1">MOD_ARTICLES_CATEGORY_OPTION_INCLUSIVE_VALUE</option>
 					<option value="0">MOD_ARTICLES_CATEGORY_OPTION_EXCLUSIVE_VALUE</option>
 				</field>
 
 				<field
 					name="created_by_alias"
 					type="sql"
-					multiple="true" size="5"
+					multiple="true"
+					size="5"
```
Comment (andrepereiradasilva) [inline modules/mod_articles_category/mod_articles_category.xml]: move multiple and size down?

File: modules/mod_articles_category/mod_articles_category.xml [hunk 3]
```diff
@@ -217,49 +230,60 @@
 					default="a.created"
 					label="MOD_ARTICLES_CATEGORY_FIELD_DATERANGEFIELD_LABEL"
```
Comment (andrepereiradasilva) [inline modules/mod_articles_category/mod_articles_category.xml]: move default down?

**Comment by andrepereiradasilva** [comment]
I have tested this item :white_check_mark: successfully on 30d0e15d7676ad891b4725aa567bc6bc72474f72<br /><br />on code review<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10573">issues.joomla.org/joomla-cms/10573</a>.</sub>

**Comment by JoomliC** [comment]
I have tested this item :white_check_mark: successfully on 30d0e15d7676ad891b4725aa567bc6bc72474f72<br /><br />On code review<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10573">issues.joomla.org/joomla-cms/10573</a>.</sub>

**Comment by brianteeman** [comment]
RTC <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10573">issues.joomla.org/joomla-cms/10573</a>.</sub>

**Pull request closed without merging**